src/Widgets/TerminalWidget.vala

```diff
@@ -42,12 +42,13 @@ namespace Terminal {
             }
         }
 
-        private Gtk.Menu menu {
+        private Menu context_menu_model {
```

Comment:
Hm so I get a warning here about:
```
../src/Widgets/TerminalWidget.vala:45.9-45.39: warning: Terminal.TerminalWidget.context_menu_model hides inherited property `Vte.Terminal.context_menu_model'. Use the `new' keyword if hiding was intentional
   45 |         private Menu context_menu_model {
```

I'm actually not sure what is the purpose of this property? it seems to only be used in a single private function

Reply:
Now removed.